.github/workflows/unit_tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       run: |
         go get gotest.tools/gotestsum@v0.4.2
         # TODO: validate bin/protoc-go.sh does not dirty the repo
-        gotestsum -- -cover -coverprofile=coverage.out -race -v -mod=readonly ./...
+        LINKERD_TEST_PRETTY_DIFF=1 gotestsum -- -cover -coverprofile=coverage.out -race -v -mod=readonly ./...
```

Comment:
not strictly related to this change but it was impossible to understand the diff without this